src/Components/Library/BldrsTabs.jsx

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable no-magic-numbers */
+import React, {useState} from 'react'
+import {styled} from '@mui/material/styles'
+import Box from '@mui/material/Box'
+import Tabs from '@mui/material/Tabs'
+import Tab from '@mui/material/Tab'
+
+
+const CustomTabs = styled(Tabs)(({theme}) => ({
+  'borderBottom': `1px solid ${theme.palette.primary.main}`,
+  '& .MuiTabs-indicator': {
+    backgroundColor: theme.palette.secondary.main,
+  },
+}))
+
+
+const CustomTab = styled((props) => <Tab disableRipple {...props}/>)(({theme}) => ({
+  'textTransform': 'none',
+  'minWidth': 0,
+  [theme.breakpoints.up('sm')]: {
+    minWidth: 0,
+  },
+  'fontSize': '1em',
+  'fontWeight': theme.typography.fontWeight,
+  'marginRight': theme.spacing(0),
+  'color': theme.palette.primary.contrastText,
+  'fontFamily': theme.typography.fontFamily,
+  '&:hover': {
+    color: theme.palette.secondary.main,
+  },
+  '&.Mui-selected': {
+    color: theme.palette.secondary.main,
+    fontWeight: theme.typography.fontWeight,
+  },
+  '&.Mui-focusVisible': {
+    backgroundColor: 'green',
+  },
+  '@media (max-width: 700px)': {
+    fontSize: '.7em',
+  },
+}))
+
+
+/**
+ * Styled Tabs component.
+ *
+ * @property {Array} tabs array of tabs
+ * @return {React.Component}
+ */
+export default function BldrsTabs({tabNames, actionCb}) {
```

Comment:
The `actionCb` argument is not used and should be removed.

Reply:
I foresee the need to pass in the callback to the tabs that exposes the value of the selected tab to other components.
For example to drive the title of the dialog. 
The call back needs to accept the value and needs to be triggered when the tab is selected.

Reply:
I think that can be added at the time it's needed in order to give you some more room for designing the interface.

Specifically, it's unclear what value would be passed back. The tab index? The tab name? Both?